src/Sarif.UnitTests/JsonTests.cs

```diff
@@ -12,14 +12,6 @@ namespace Microsoft.CodeAnalysis.Sarif
 {
     public abstract class JsonTests
     {
-        private const SarifVersion CurrentVersion = SarifVersion.TwoZeroZero;
-
-        protected static readonly string SarifSchemaUri =
-            SarifUtilities.ConvertToSchemaUri(CurrentVersion).OriginalString;
-
-        protected static readonly string SarifFormatVersion =
-            SarifUtilities.ConvertToText(CurrentVersion);
```

Comment:
>; [](start = 56, length = 1)

Moved these lines to `SarifUtilities` so I could use them in the Multitool unit tests as well as the Sarif unit tests. I could have moved them to test utilities, but really they don't seem test-specific to me; they seem useful in product code.